src/core/control/XournalMain.cpp

```diff
@@ -158,47 +170,36 @@ void checkForErrorlog() {
     std::sort(errorList.begin(), errorList.end());
     std::string msg =
             errorList.size() == 1 ?
-                    _("There is an errorlogfile from Xournal++. Please send a Bugreport, so the bug may be fixed.") :
-                    _("There are errorlogfiles from Xournal++. Please send a Bugreport, so the bug may be fixed.");
+                    _("There is an errorlogfile from Xournal++. Please fill a Bugreport, so the bug may be fixed.") :
+                    _("There are errorlogfiles from Xournal++. Please fill a Bugreport, so the bug may be fixed.");
```

Comment:
I guess you wanted to write it like this:
```suggestion
                    _("There is an errorlogfile from Xournal++. Please file a Bugreport, so the bug may be fixed.") :
                    _("There are errorlogfiles from Xournal++. Please file a Bugreport, so the bug may be fixed.");
```